experimental/HttpBenchmark/HttpBenchmark.csproj

```diff
@@ -10,7 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.CommandLineUtils.Sources" Version="1.2.0-*" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.Extensions.CommandLineUtils.Sources" Version="2.0.0-*" PrivateAssets="All" />
```

Comment:
Suggest putting versions in a .props file like https://github.com/aspnet/KestrelHttpServer/blob/dev/build/dependencies.props